apps/nextjs-website/src/lib/webinars/dynamodb/codec.ts

```diff
@@ -34,11 +38,19 @@ export const makeWebinarQuestionFromDynamodbItem = (
   webinarId: input.webinarId.S,
   question: input.question.S,
   createdAt: input.createdAt.S,
+  hiddenBy: input.hiddenBy.S,
+  hiddenByFullName: input.hiddenByFullName.S,
+  highlightedBy: input.highlightedBy.S,
+  highlightedByFullName: input.highlightedByFullName.S,
```

Comment:
```suggestion
  hiddenBy: input.hiddenBy.S,
  highlightedBy: input.highlightedBy.S,
```
same as another comment, I'd keep only the attributes we need to persist according the requirements